src/components/TestPage/TestPage.jsx

```diff
@@ -109,8 +108,11 @@ export const TestPage = () => {
     }
   };
 
-  const onKeyPress = () => {
-    if (state.time.m === 1) {
+  const onKeyPress = (e) => {
+    if (
+      (state.time.m === 1 && e.which >= 65 && e.which <= 90) ||
```

Comment:
It might be better to check if  `AppStates` is `Initial` instead of `state.time.m === 1`